### DISCUSSION TITLE
Couldn't push from my local machine to remote repo

### DISCUSSION THREAD

**Discussion by dezmanvee** [community/community#49101] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm not clear on why I can't push my project from my local PC to GitHub.

I created a GitHub repository, and then on my local PC I set the remote using git remote add origin git@github.com:dezmanvee/fotma-repo.git.  Afterwards I added, commited, and tried pushing my changes using _git push -u origin main_. I get the following error:

![fatal error](https://user-images.githubusercontent.com/105369479/222796351-a4ad395b-6e08-4f47-9ae0-b2ad67764075.PNG)

**Comment by tyler-holm** [comment]
Likely a typo when setting the remote. Can you check the remote url and verify that its correct?

If the url looks ok then make sure you ran all of the following commands. including `git branch -M main`

```
git add .
git commit -m "first commit"
git branch -M main
git remote add origin git@github.com:dezmanvee/fotma-repo.git
git push -u origin main
```

If you ran all of those and it's still not working then you'll have to delete your .git folder and redo `git init` followed by all the commands I listed.
